test/new-e2e/examples/agentenv_file_permissions_test.go

```diff
@@ -88,8 +89,8 @@ func (v *filePermissionsTestSuite) TestUserGroupPermissions() {
 
 func (v *filePermissionsTestSuite) TestSecretsPermissions() {
 	files := []agentparams.Option{
-		agentparams.WithFileWithPermissions("/tmp/secrets", "typical secret config", false, perms.NewUnixPermissions(perms.WithPermissions("0700"), perms.WithOwner("dd-agent"), perms.WithGroup("dd-agent"))),
-		agentparams.WithFileWithPermissions(`/tmp/secrets_root_group`, "secret config for security agent", false, perms.NewUnixPermissions(perms.WithPermissions("0750"), perms.WithOwner("dd-agent"), perms.WithGroup("root"))),
+		agentparams.WithFileWithPermissions("/tmp/secrets", "typical secret config", false, secrets.WithUnixSecretPermissions(false)),
```

Comment:
I think you have the wrong target branch for your PR, I couldn't find `WithUnixSecretPermissions`

Reply:
Yep, base branch is wrong, I'll rebase main when https://github.com/DataDog/datadog-agent/pull/24723 is merged